src/etc/inc/plugins.inc.d/unbound.inc

```diff
@@ -486,12 +486,12 @@ function unbound_add_domain_overrides($pvt = false)
                 $domain_entries .= "local-zone: \"$domain\" typetransparent\n";
             }
         } else {
-            $domain_entries .= "stub-zone:\n";
+            $domain_entries .= "forward-zone:\n";
             $domain_entries .= "\tname: \"$domain\"\n";
             foreach ($ips as $ip) {
-                $domain_entries .= "\tstub-addr: $ip\n";
+                $domain_entries .= "\tforward-addr: $ip\n";
             }
-            $domain_entries .= "\tstub-prime: no\n";
+            //$domain_entries .= "\tstub-prime: no\n";
```

Comment:
it's better to remove stale code. we always have git history :)

Reply:
That is very true.